CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.9.0 (2020-03-20)
+
+- Fixed the datepicker component to not error when invalid numeric dates are entered. [#127](https://github.com/blackbaud/skyux-datetime/pull/127)
```

Comment:
Fixed the datepicker component to prevent errors when invalid numeric dates are entered.

Reply:
Should `errors` be `an error` in the rewrite? Looking at the related issue, it seems like we're trying to prevent one particular error.

Reply:
Yup. I have updated this for both suggestions here.